jfconnect/manager.go

```diff
@@ -0,0 +1,45 @@
+package jfconnect
+
+import (
+	"github.com/jfrog/jfrog-client-go/config"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	"github.com/jfrog/jfrog-client-go/jfconnect/services"
+)
+
+type Manager interface {
+	LogMetric([]byte) error
+}
+
+type jfConnectManager struct {
+	client *jfroghttpclient.JfrogHttpClient
+	config config.Config
+}
+
+func NewManager(config config.Config) (Manager, error) {
+	details := config.GetServiceDetails()
+	var err error
+	manager := &jfConnectManager{config: config}
+	manager.client, err = jfroghttpclient.JfrogClientBuilder().
+		SetCertificatesPath(config.GetCertificatesPath()).
+		SetInsecureTls(config.IsInsecureTls()).
+		SetClientCertPath(details.GetClientCertPath()).
+		SetClientCertKeyPath(details.GetClientCertKeyPath()).
+		AppendPreRequestInterceptor(details.RunPreRequestFunctions).
+		SetContext(config.GetContext()).
+		SetDialTimeout(config.GetDialTimeout()).
+		SetOverallRequestTimeout(config.GetOverallRequestTimeout()).
+		SetRetries(config.GetHttpRetries()).
+		SetRetryWaitMilliSecs(config.GetHttpRetryWaitMilliSecs()).
+		Build()
+
+	return manager, err
+}
+
+func (jm *jfConnectManager) Client() *jfroghttpclient.JfrogHttpClient {
+	return jm.client
+}
+
+func (jm *jfConnectManager) LogMetric(metric []byte) error {
```

Comment:
The name LogMetric doesn’t convey that it’s performing a send action; it sounds more like a getter method (Log being interpreted as a noun rather than an action).

Reply:
I changed it to `PostMetric`.